src/main/java/mate/academy/application/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.application.exception;
+
+public class EntityNotFoundException extends RuntimeException {
+    public EntityNotFoundException(String message) {
+        super(message);
+    }
+}
```

Comment:
EntityNotFoundException already exists in JPA, please use the existing one instead of implementing your own